descriptors/ModuleDescriptor-template.json

```diff
@@ -638,6 +638,28 @@
       ],
       "visible": true
     },
+    {
+      "permissionName": "consortia.sharing-roles-all.item.post",
+      "displayName": "Consortia manager: Share roles",
+      "description": "Consortia manager: Share roles",
```

Comment:
small suggestion: maybe Share roles All or smth else as different name for new permision

Reply:
Display name gathered from story acceptance criterias. This permission will be visible on UI, all other sharing roles permissions are not displayed on UI